sass/DynamicFacet/_DynamicFacetHeader.scss

```diff
@@ -15,6 +15,7 @@
   white-space: nowrap;
   text-overflow: ellipsis;
   overflow: hidden;
+  text-transform: capitalize;
```

Comment:
some localized texts are lowercased, this will ensure that no title is inconsistent in the future.